src/components/Activities/Activity3/Game/FinalReport.tsx

```diff
@@ -60,8 +61,13 @@ function FinalReport(props: FinalReportProps): JSX.Element {
               <div className='bar' style={{ background: getGradient(featureWeights) }}>
                 {featureWeights.map((t, i) =>
                   <>
-                    <div data-tip data-for={`variable-${i}`} key={i} style={{ width: `${t}%`, height: '30px' }}></div>
-                    <ReactToolTip id={`variable-${i}`}>{t}%</ReactToolTip>
+                    {t > PERCENT_THRESHOLD.VARIABLES ? //percentage where the text no longer fits inside the corresponding section, note this is not a pixel value, and rather a percentage based on my laptop screen
```

Comment:
nit: i dont think we need this here since its part of the info for `PERCENT_THRESHOLD` rather than just inlining it random code that uses it. if we do the change as per below, then the explanation will appear on hover

```suggestion
                    {t > PERCENT_THRESHOLD.VARIABLES ?
```

Reply:
Yea your right. I originally had magic numbers so I had the comments in the code, but I decided to move them to a const variable, and I wasn't sure if I should remove the comment or not.